### ISSUE TITLE
The description of the "Projection" plugin is incorrect before installing the plugin 

### ISSUE THREAD

**Issue by sebastien-rosset** [OpenCPN/plugins#1169]
# Problem Description
In Settings -> Plugins, the description of the "Projection" plugin is wrong **before** installing the plugin. The description states "NMEA0183 to SignalK Converter".

<img width="760" alt="image" src="https://github.com/user-attachments/assets/81460667-fec2-4d59-9988-0cd650e7f3a0">

# Additional Context

After installing the plugin, the description is correct:
"Projection Plugin offers alternate projections".

<img width="762" alt="image" src="https://github.com/user-attachments/assets/c571073d-9a39-484c-a854-040e41b2617e">

**Comment by rgleason** [comment]
@sebastien-rosset  This is the current repo for projections  Pavel [Nohal]
https://github.com/nohal/projections_pi

**Comment by sebastien-rosset** [comment]
> @sebastien-rosset This is the current repo for projections Pavel [Nohal] https://github.com/nohal/projections_pi

Yes, I created this PR: https://github.com/nohal/projections_pi/pull/1
Unfortunately, @nohal has disabled issues for that repo.

**Comment by rgleason** [comment]
Ok got it. Thanks.

**Comment by nohal** [comment]
While I have made the build for current OpenCPN on request, my plate is full and I do not intend to take over this plugin as a maintainer, that is why issues are not enabled in my fork, all I can do is create a pull request to the (unmaintained) upstream.

**Comment by rgleason** [comment]
Ok Sean Depagnier has an early version of this plugin and his Issues is open
https://github.com/seandepagnier/projections_pi/issues

This issue can be placed there.  Also I have an Projections repos too. I did not want to step on toes here Pavel, so I will pull your work to my repos and will try to maintain it if you want. 

At least we know that Issues have a home at sean's older repos. https://github.com/seandepagnier/projections_pi/issues

Sebastien, would you mind moving your Issue there?  Let me know if you want me to do it. Thanks.
